backends/platform/sdl/opendingux/miyoo.mk

```diff
@@ -0,0 +1,46 @@
+OD_EXE_STRIPPED := scummvm_stripped$(EXEEXT)
+bundle = sd-root
+
+all: $(OD_EXE_STRIPPED)
+
+$(OD_EXE_STRIPPED): $(EXECUTABLE)
+	$(STRIP) $< -o $@
+
+$(bundle): all
+	$(MKDIR) -p $(bundle)/games/scummvm
+	$(CP) $(DIST_FILES_DOCS) $(bundle)/games/scummvm
+	$(MKDIR) $(bundle)/games/scummvm/themes
+	$(CP) $(DIST_FILES_THEMES) $(bundle)/games/scummvm/themes/
+
+ifdef DIST_FILES_ENGINEDATA
+	$(MKDIR) $(bundle)/games/scummvm/engine-data
+	$(CP) $(DIST_FILES_ENGINEDATA) $(bundle)/games/scummvm/engine-data/
+ifeq ($(OPENDINGUX_TARGET), rg99)
```

Comment:
This looks like a leftover from the OpenDingux bundle rules

Reply:
Removed. Thanks